### PR TITLE
marsRover37: Adds a navigation bar to the top of the page

### PR DESCRIPTION
**Pull request by HeatherJLewis**
**Description**
Add a navigation menu

**Changes**
* adds navigation and styling on `/` and `/login`

### CODE REVIEW

**Comment by HeatherJLewis** [comment]
How do we go about removing the repetition for this?
Search results suggest using php - but I'm not sure about this at all.
Could we create a server side template for the nav bar and inject it into the HTML by fetching it from an endpoint (`/navBar` for instance)?
I'm put off this because I think I'd need to use `innerHTML` (or could I use `textContent`).
Also, I think I'd need to add pug and a pre-processor?
Also, how would this affect page loading performance?
Too many questions :)

File: app/homepage/index.html [hunk 1]
```diff
@@ -7,6 +7,13 @@
         <link rel="stylesheet" href="./styles.css">
     </head>
     <body>
+        <ul>
+            <li><a class="active" href="/">Home</a></li>
+            <li><a href="/user/rover">Rover</a></li>
+            <li><a href="/calculator">Calculator</a></li>
+            <li><a href="/login">Login</a></li>
+            <li><a href="/register">Register</a></li>
+        </ul>
```
Comment (Kenneth-Siu) [inline app/homepage/index.html:16]: A spot of semantic HTML here, perhaps? `<nav>`

File: app/homepage/styles.css [hunk 2]
```diff
@@ -14,15 +15,36 @@ body {
...
+
+li a:hover {
+  padding: 10px;
+  text-decoration: underline;
+}
+
+.active {
+  color: var(--main-color-one);
+  background-color: var(--main-color-two);
+}
```
Comment (Kenneth-Siu) [inline app/homepage/styles.css:39]: Given that all of these are likely to be used for only the navigation bar, perhaps we should be more specific with our CSS selectors, or else in the future any future usages of `<li>` will be met with confusion

File: app/homepage/index.html (hunk 1 above)
Comment (Kenneth-Siu) [reply app/homepage/index.html:16]: And also the same for the other pages

File: app/homepage/styles.css (hunk 2 above)
Comment (Kenneth-Siu) [reply app/homepage/styles.css:39]: And also the same for the other pages

**Pull request closed without merging**